### ISSUE TITLE
[advice] Make fzf more robust

### ISSUE THREAD

**Issue by Freed-Wu** [gennaro-tedesco/gh-f#16]
I advice change 12 `fzf`s to `fzf -d' '`s. Because fzf will be affected by `$FZF_DEFAULT_OPT`. If user use `-d:` or other `-d`, it will make fzf work abnormally.

Thanks.

**Comment by gennaro-tedesco** [comment]
Sorry for the long wait in responding to this issue, and thank you for the recommendation: definitely a good idea. I will work on it in the next release :)

**Comment by gennaro-tedesco** [comment]
[This PR](https://github.com/gennaro-tedesco/gh-f/pull/18) addresses the issue: would you be so kind to help me test it to make sure there are no particular corner cases where the changes may instead behave otherwise? Ideally you could clone the repository and install the local branch 
```
gh repo clone gennaro-tedesco/gh-f
git checkout fzf_robust
gh extension remove f
gh extension install .
```
and use it off: so far it seems all is good, but four eyes are better than two!

**Comment by Freed-Wu** [comment]
Thanks. I have used this version, and since now I haven't found any incorrect behaviour.

**Comment by gennaro-tedesco** [comment]
Released in [v0.3.3](https://github.com/gennaro-tedesco/gh-f/releases/tag/v0.3.3). Just `gh extension upgrade f`.
